src/swagger/schemas/monsters.yml

```diff
@@ -228,7 +282,9 @@ monster-model:
             - unaligned
         armor_class:
           description: 'The difficulty for a player to successfully deal damage to a monster.'
-          type: number
+          type: array
+          items:
+            $ref: '#/monster-armor-class'
```

Comment:
Don't forget to update the example monster under `swagger/paths/monsters.yml` as well.